thousandeyes/util.go

```diff
@@ -122,6 +135,36 @@ func ResourceBuildStruct(d *schema.ResourceData, structPtr interface{}) interfac
 	return resourceFixups(d, structPtr)
 }
 
+// resourceRead is a generic function for reading resources.
```

Comment:
nit: generic function for reading **resources** or just **tests**? we're logging "ThousandEyes Test" inside.

Reply:
Will change to reading resource ;)